consensus/ibft/sign.go

```diff
@@ -43,7 +43,7 @@ func ecrecoverFromHeader(h *types.Header) (types.Address, error) {
 }
 
 func signSealImpl(prv *ecdsa.PrivateKey, h *types.Header, committed bool) ([]byte, error) {
-	sig, err := signHash(h)
+	sig, err := getHeaderHash(h)
```

Comment:
Please change variable sig to a more proper name, since it's not a signature:
```suggestion
	hash, err := getHeaderHash(h)
```